automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -43,7 +43,7 @@ groups:
       description: "No new blocks posted to the google storage bucket for {{ $labels.testnet }}."
 
   - alert: ProverErrors
-    expr: max by (testnet) (max_over_time(Coda_watchdog_prover_errors_total ${rule_filter} [${alerting_timeframe}])) >= 0
+    expr: max by (testnet) (max_over_time(Coda_watchdog_prover_errors_total ${rule_filter} [${alerting_timeframe}])) = 0
```

Comment:
I don't know this language, but should the `=` be `>`, given the PR description?

Reply:
Yes, I just messed up the edit. I edited it correctly in the grafana UI to test, but screwed up this edit. Thanks for catching!